feedback/src/webapp/WEB-INF/bootstrap.jsp

```diff
@@ -78,15 +78,11 @@
         <div class="portletBody">
             <ul id="feedback-toolbar" class="navIntraTool actionToolBar" role="menu"></ul>
             <div class="portletBody">
-                <div id="feedback-error-message-wrapper">
-                    <div>
-                        <span></span><a href="javascript:;" alt="${i18n.close}" title="${i18n.close}"></a>
-                    </div>
+                <div id="feedback-error-message-wrapper" class="sak-banner-error">
+                    <span></span>
```

Comment:
should the link be taken out?

Reply:
We don't have a working dismissible banner component defined yet, although it is on the list for SWITCH. 

For now, it matches other tools to have the message stay on the page until the page is refreshed.

Reply:
Exactly, I think the link should be added globally, not at a tool level.